package.yaml

```diff
@@ -5,7 +5,7 @@
 spec-version: 0.31.0
 
 name:                xrefcheck
-version:             0.1.1.2
+version:             0.1.1.3
```

Comment:
I am not sure but I think when you add some user-visible functionality you should update minor version number (i. e. the third one).

We don't explicitly follow PVP, but I'd use it as guidance: https://pvp.haskell.org/

It says:
> Non-breaking change. Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then A.B MAY remain the same but the new C MUST be greater than the old C. Note that modifying imports or depending on a newer version of another package may cause extra non-orphan instances to be exported and thus force a minor version change.

That's more applicable to libraries while our package is unlikely to be used as a library. But if we extend it to executables, I think the adding of a new command line option fits into this point and hence we should increase C.

Reply:
Right. A bit surprising that we got so many releases (2 = "many" today) which do not change anything significant.